group-generators/generators/tornado-cash-depositors/index.ts

```diff
@@ -0,0 +1,125 @@
+import { dataOperators } from "@group-generators/helpers/data-operators";
+import {BigQueryProvider} from "@group-generators/helpers/data-providers/big-query/big-query";
+
+import {
+  ValueType,
+  Tags,
+  FetchedData,
+  GroupWithData,
+  AccountSource,
+} from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Daily,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const bigQueryProvider = new BigQueryProvider();
+
+    // Mainnet Torando Cash contract address: 0.1ETH
+    const tornadoCashDepositors01ETH = "0x12D66f87A04A9E220743712cE6d9bB1B5616B8Fc";
+    // Mainnet Torando Cash contract address: 1ETH
+    const tornadoCashDepositors1ETH = "0x47CE0C6eD5B0Ce3d3A51fdb1C52DC66a7c3c2936";
+    // Mainnet Torando Cash contract address: 10ETH
+    const tornadoCashDepositors10ETH = "0x910Cbd523D972eb0a6f4cAe4618aD62622b39DbF";
+    // Mainnet Torando Cash contract address: 100ETH
+    const tornadoCashDepositors100ETH = "0xA160cdAB225685dA1d56aa342Ad8841c3b53f291";
+
+    const depositFunctionEthABI =
+    "function deposit(bytes32 _commitment) external payable nonReentrant";
+    type DepositFunction = {
+    _commitment: string;
+    };
+
+    // Get all 0.1ETH Deposits on Tornado Cash
+    const getTornadoCash01DepositTransactions =
+    await bigQueryProvider.getAllTransactionsForSpecificMethod<DepositFunction>(
+      {
+        functionABI: depositFunctionEthABI,
+        contractAddress: tornadoCashDepositors01ETH,
+        options: {
+          functionArgs: true,
```

Comment:
I am not sure that you need functionArgs property set to true since you only focus on the from property in the transaction